official/resnet/resnet_run_loop.py

```diff
@@ -565,7 +568,7 @@ def define_resnet_flags(resnet_size_choices=None):
       help=flags_core.help_wrap('Skip training and only perform evaluation on '
                                 'the latest checkpoint.'))
   flags.DEFINE_boolean(
-      name="image_bytes_as_serving_input", default=True,
+      name="image_bytes_as_serving_input", default=False,
```

Comment:
Any particular reason the default is set to false?

Reply:
Two reasons:
1) This doesn't work properly on CIFAR, so users would have to know to go and set this to False.
2) This is a finesse in that the input is no longer just a normal tensor, and I am generally wary of putting advanced use cases as defaults.

Reply:
I see, thanks for the response